test/blackbox-tests/test-cases/variants/run.t

```diff
@@ -202,3 +202,25 @@ Implementations may refer to virtual library's modules
           test alias default
   bar from vlib
   Foo.run implemented
+
+Implementations may have private modules that have overlapping names with the
+implementation lib
```

Comment:
Shouldn't this be:
```suggestion
virtual lib
```

Reply:
I'm loving this github feature already.

Reply:
Yh, I was pleasantly surprised by it this morning